packages/frontend/core/src/components/affine/setting-modal/general-setting/plans/lifetime/believer-card.css.ts

```diff
@@ -71,12 +77,18 @@ export const bg = style({
   },
 });
 globalStyle(`.${bg} > svg.affine-svg`, {
-  color: colorSchemes.light.affine,
+  vars: {
+    '--start': colorSchemes.light.affine.start,
```

Comment:
I am a little worried if it is hard to maintain the connection between `--start` with the svg in lifetime/assets.ts.
At least please leave some informative comments here.

Reply:
Replaced with `createVar()`, and I think it should be better.